spec/models/form/question_spec.rb

```diff
@@ -357,8 +357,8 @@
       end
 
       context "when Sales form" do
-        let(:sales_log) { FactoryBot.create(:sales_log, :completed, ethnic_group: 17, saledate: Time.zone.local(2022, 1, 1)) }
-        let(:question) { sales_log.form.get_question("ethnic_group", sales_log) }
+        let(:sales_log) { FactoryBot.create(:sales_log, :completed, national: 13, saledate: Time.zone.local(2022, 1, 1)) }
+        let(:question) { sales_log.form.get_question("national", sales_log) }
```

Comment:
Why do we need this change?

Reply:
This is a test about inferred check answers values which the ethnic_group question no longer has - I just picked another question that still has one.